website/.gitignore

```diff
@@ -20,6 +20,6 @@ yarn-debug.log*
 yarn-error.log*
 
 # OpenAPI docusaurus generated stuff
-docs/reference/apis/**/sidebar.js
+docs/reference/api/**/sidebar.js
```

Comment:
This is due to a last-minute change in where we generate OpenAPI docs. This change simply makes sure the new generated docs get ignored.